docs/execution.rst

```diff
@@ -29,14 +29,20 @@ The result of a query is an ``ExecutionResult`` which is the query data and/or a
 
```

Comment:
Ahh must have missed this doco update.  I think this is on master so I am unsure why this comes up as a diff